guides/computation-development.md

```diff
@@ -0,0 +1,400 @@
+# Computation Development
+
+_This guide will walk through basic algorithm development in the COINSTAC ecosystem. It is intended for algorithm authors and developers._
+
+## Basic overview of COINSTAC system
+
+COINSTAC is an ecosystem for running decentralized algorithms with many clients. It uses a client-server model: while arbitrary calculations can run on clients and servers, the system’s primary use case is differential privacy, where clients run algorithms with data to ensure their privacy. Clients submit their anonymized results to the server, which amalgamates clients’ data and performs interesting computations, such as averaging, modeling, etc.
+
+COINSTAC’s core is written entirely in JavaScript; writing an algorithm requires writing an entry point in this language.
+ However, the system is robust enough to support algorithms in any language. This guide will explain
```

Comment:
Haha, oh yikes. Missed that. Dropped phrase in 7428904e047d7627b37efad206754c2ee1015579.

Perhaps we need a table of contents?